LLVM_FOR_RISCV.md

```diff
@@ -158,15 +181,3 @@ Linux:
 ```bash
 [PATH_TO_YOUR_LLVM_LINUX_INSTALL]/bin/clang --target=riscv64-unknown-linux-gnu -o hello hello.c
```

Comment:
isnt this path known due to completing the install?

Reply:
it is, but it's just an example, after changes to script this will be given by user